concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -213,26 +211,20 @@ def main():
         PROVIDER_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
         force_subtask_results(
-            timestamp = timestamp_to_isoformat(current_time),
-            ack_report_computed_task =ack_report_computed_task(
-                timestamp = timestamp_to_isoformat(current_time - 28801),  # current_time - 08:00:01
-                subtask_id = task_id + '2C',
-                task_to_compute = task_to_compute(
-                    timestamp = timestamp_to_isoformat(current_time - 28802),  # current_time - 08:00:02
-                    compute_task_def = compute_task_def(
-                        task_id     = task_id + '2C',
-                        subtask_id  = task_id + '2C',
-                        deadline    = current_time,
-                    )
-                )
+            timestamp                   = timestamp_to_isoformat(current_time),
+            ack_report_computed_task    = ack_report_computed_task(
+                timestamp               = timestamp_to_isoformat(current_time - 2 * cluster_consts.subtask_verification_time - TIME_GUARD),  # current_time - 08:00:10
+                subtask_id              = task_id + '2C',
+                task_to_compute         = create_task(timestamp_to_isoformat(current_time - 28802), task_id + '2C', current_time)
```

Comment:
Line is too long, each parameter should be in separate line.

Reply:
done;